### ISSUE TITLE
bug v3: missing Trouble commands

### ISSUE THREAD

**Issue by illya-laifu** [folke/trouble.nvim#422]
### Did you check docs and existing issues?

- [X] I have read all the trouble.nvim docs
- [X] I have searched the existing issues of trouble.nvim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

0.9.5 Release

### Operating system/version

EndeavorOS x86_64

### Describe the bug

Editor shows an error `Not an editor command: Trouble` when executing Trouble from command mode, the command is not listed in `:commands`

### Steps To Reproduce

1. Execute `:Trouble`, `:Trouble diagnostics toggle`, etc.

### Expected Behavior

Visible Trouble buffer.

### Repro

```Lua
-- DO NOT change the paths and don't remove the colorscheme
local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({ "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", lazypath, })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
  "folke/tokyonight.nvim",
  { "folke/trouble.nvim", branch = "dev" },
  -- add any other plugins here
}
require("lazy").setup(plugins, {
  root = root .. "/plugins",
})

vim.cmd.colorscheme("tokyonight")
-- add anything else here
```

**Comment by illya-laifu** [comment]
Found the issue. Adding `opts = {}` or manually calling `.setup()` registers the commands. I was under (wrong) assumption, that omitting `opts` would fallback to the default config.

**Comment by githorse** [comment]
I was just bit by this one too. It would certainly be gentler (and less surprising) to make `opts` optional rather than requiring `opts = {}`. I wasted some time trying to figure out what's going on here since `lazy.nvim` tells me `Trouble` is loaded just fine.

**Comment by jonathanmorris180** [comment]
Also had this issue—fixed by adding `opts = {}`

**Comment by folke** [comment]
If you'd all just read/follow the install instructions, you wouldn't have this issue.

**Comment by tugglecore** [comment]
Firstly, thanks for the awesome plugin!

I am using [vim-plug](https://github.com/junegunn/vim-plug) so my comment will be most helpful to other users of [vim-plug](https://github.com/junegunn/vim-plug). 

I encountered this issue today and resolved it by calling `setup` for `Trouble`. For me, I added the following line to my Lua install script: `require('Trouble').setup()`. 

I believe installing versions of `trouble<3.0.0` automatically set up `Trouble` commands. I think this is not true for `Trouble>=3.0.0` so just installing `Trouble` will not set up any commands.

I hope this helps someone.
